scripts/install_basics.sh

```diff
@@ -18,6 +18,15 @@ conda install -y python=3.8
 # install unittest-xml-reporting
 pip install unittest-xml-reporting
 
+# install CUDA 11.6
```

Comment:
Why do we need CUDA 11.6 for this reading from S3 change?

Reply:
This is to fix the Circle CI test error, seems 11.3 is not supported anymore.